packages/react-reconciler/src/ReactFiberCommitHostEffects.js

```diff
@@ -214,6 +218,25 @@ function getHostParentFiber(fiber: Fiber): Fiber {
   );
 }
 
+export function getFragmentInstanceParent(
+  fiber: Fiber,
+): null | FragmentInstance {
```

Comment:
It seems to me that there can be more than one parent Fragment. When I wrap a Component I shouldn't have to know whether or not there's another Fragment wrapping it. It seems like it's set up now, adding a Fragment would block parents from inspecting the insides - even by accident.